pyteal/ast/frame.py

```diff
@@ -89,19 +89,15 @@ def __init__(
                 f"be greater than local allocations {len(local_stack_types)}."
             )
 
-        if not all(map(lambda t: t != TealType.none, arg_stack_types)):
+        if not all(
+            map(lambda t: t != TealType.none, arg_stack_types + local_stack_types)
+        ):
```

Comment:
```suggestion
        if any(t == TealType.none for t in arg_stack_types + local_stack_types):
```

Reply:
👍

Reply:
FYI - I just tried out OpenAI ChatGPT with this refactoring to see if this could have been automated. Low and behold:

@ahangsu @michaeldiamant @jasonpaulos 

<img width="859" alt="image" src="https://user-images.githubusercontent.com/291133/205493456-ed8ab13a-3306-401b-8bd8-c33542289fd6.png">